doc/developer-notes.md

```diff
@@ -622,13 +622,16 @@ E.g. in member initializers, prepend `_` to the argument name shadowing the
 member name:
 
 ```c++
-class AddressBookPage
+struct AddressBookPage
 {
-    Mode m_mode;
```

Comment:
members should start with `m_` according to the developer notes

Reply:
>    - Class member variables have a `m_` prefix.

This is now a `struct`.

Reply:
Structs and classes are the same thing in C++. Only default visibility is different.